frontend/src/components/common/ToastMessage.vue

```diff
@@ -75,6 +76,6 @@ app.config.errorHandler = (err, instance, info) => {
 
 <template>
     <Toast group="tl" position="top-right" #icon>
-        <ErrorFilledIcon class="iconTest"></ErrorFilledIcon>
+        <ErrorFilled />
```

Comment:
Same comment as previous, maybe I am confused, is it better to use Icon component?